app/views/SidebarView/index.js

```diff
@@ -153,7 +153,7 @@ export default class Sidebar extends Component {
 			const permissionsFiltered = database.objects('permissions')
 				.filter(permission => permissions.includes(permission._id));
 			return permissionsFiltered.reduce((result, permission) => (
-				result || permission.roles.some(r => roles.includes(r))),
+				result || permission.roles.some(r => roles.indexOf(r) !== -1)),
```

Comment:
why?

Reply:
I don't know but ```includes``` cause an error, I tried to change this based on this issue:
https://github.com/facebook/create-react-app/issues/942